### ISSUE TITLE
docs: Issue on page Configuration Parameters task_ttl_in_seconds

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#19040]
I would like to report an issue on page https://opensource.docs.scylladb.com/master/reference/configuration-parameters.html#confval-task_ttl_in_seconds

### Problem

The description (below), extracted from the code, of `task_ttl_in_seconds` does not explain what this timer is used for
Is it Alternator TTL? CQL? How is this value used? What zero means?
> Time for which information about finished task stays in memory.

###  Suggest a fix
More elaborate doc

**Comment by tzach** [comment]
@nyh can you help?
(this is a followup to https://github.com/scylladb/scylladb/issues/16714 )

**Comment by nyh** [comment]
No, this has nothing to do with either Alternator's or CQL's TTL. As "git blame" can show, this configuration option was added  by @Deexie in commit 2439e559742 (August 2022), and determines how long to keep finished "tasks" in memory to allow the user to investigate them.

I agree we have two many uses of the word "TTL" and we should be clearer and also have documentation for the entire task-tracking feature and point to this documentation from individual configuration options that belong to this feature.
